CONTRIBUTORS.md

```diff
@@ -49,7 +49,7 @@ conda activate conversant
 
 Install `poetry`:
 ```
-pip install poetry==1.3.2
+pip install poetry==1.2.2
```

Comment:
oh damn good catch -- IIRC this was when I confused the poetry and poetry-core versions earlier